Stop over card

jonastnl opened the pull request:
Added Stopover_Card and Current_Location_Card to show on Homepage

jonastnl committed:
Added stopoverCard prototype

jonastnl committed:
extracted StopoverCard to file

jonastnl committed:
added Current_location card with edit button

jonastnl committed:
deleted import

jonastnl committed:
Merge branch 'feat/destination_screens' into stopOverCard

MhouneyLH commented on lib/presentation/widgets/current_location_card.dart:
```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/material.dart';
+
+class CurrentLocationCard extends StatefulWidget {
+  final String startStation;
```
Durch StationEntity austauschen und die Werte verwenden.

MhouneyLH replied:
auf Basis derer Werte die Properties in der Card anpassen

MhouneyLH commented on lib/presentation/widgets/current_location_card.dart:
```diff
@@ -0,0 +1,43 @@
+}
+
+class _CurrentLocationCardState extends State<CurrentLocationCard> {
+  // Hier kannst du Zustandsvariablen definieren und verwalten.
```
weg mit dem Kommentar

MhouneyLH commented on lib/presentation/widgets/stopover_card.dart:
```diff
@@ -0,0 +1,39 @@
+}
+
+class _StopoverCardState extends State<StopoverCard> {
+  // Hier kannst du Zustandsvariablen definieren und verwalten.
```
weg

MhouneyLH commented on lib/presentation/widgets/stopover_card.dart:
```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+
+class StopoverCard extends StatefulWidget {
+  final String startStation;
+  final String distance;
+  final String departureArrival;
+  final String track;
```
Hierfür die MeansOfTransportEntity verwenden und auf deren Properties zugreifen. (Da kommt noch eine StationEntity als Feld in der MeansOfTransportEntity mit dazu)

jonastnl committed:
added editTimeToStart Edit Page

to edit the time needed to go to the start location. returns a TimeOfDay type and shows the time in snackbar on showHomesPage

jonastnl committed:
added local deleteHome via Swiping the Card; CurrentLocationCard returns a TimeOfDay

jonastnl committed:
removed HomeEntity creation from FloatingActionButton
